### PR TITLE
Refactor button

### PR DESCRIPTION
**Pull request by Baribal02**
The button is now at the bottom of the screen
There is also a confirm pop up when the pilot wants to stop the flight
image

### CODE REVIEW

File: app/src/main/java/ch/epfl/skysync/MainActivity.kt [hunk 1]
```diff
@@ -22,14 +22,14 @@ import com.google.firebase.auth.FirebaseAuth
...
   private var userGlobalViewModel: UserGlobalViewModel? = null
-  private val db: FirestoreDatabase = FirestoreDatabase()
+  private val db: FirestoreDatabase = FirestoreDatabase(true)
   private val repository: Repository = Repository(db)
 
   private fun onSignInResult(result: FirebaseAuthUIAuthenticationResult) {
     val vm = userGlobalViewModel ?: return
     if (result.resultCode == RESULT_OK) {
       val user = FirebaseAuth.getInstance().currentUser!!
-      vm.loadUser(user.uid, user.email!!)
```
Comment (Plouc314) [inline app/src/main/java/ch/epfl/skysync/MainActivity.kt:32]: Looks like leftovers from debugging

File: app/src/main/java/ch/epfl/skysync/screens/crewpilot/Flight.kt [hunk 2]
```diff
@@ -228,6 +219,36 @@ fun FlightScreen(
                           .background(color = Color.White, shape = RoundedCornerShape(8.dp))
                           .padding(6.dp))
             }
+            var showConfirmationDialog by remember { mutableStateOf(false) }
```
Comment (Plouc314) [inline app/src/main/java/ch/epfl/skysync/screens/crewpilot/Flight.kt:222]: This should be moved up with the other variables

**Review by Plouc314 — APPROVED** [review]
LGTM

File: app/src/main/java/ch/epfl/skysync/MainActivity.kt (hunk 1 above)
Comment (Baribal02) [reply app/src/main/java/ch/epfl/skysync/MainActivity.kt:32]: You're right. It's now removed

File: app/src/main/java/ch/epfl/skysync/screens/crewpilot/Flight.kt (hunk 2 above)
Comment (Baribal02) [reply app/src/main/java/ch/epfl/skysync/screens/crewpilot/Flight.kt:222]: Moved it !

**Pull request merged**